cmd/uninstall.go

```diff
@@ -52,11 +52,11 @@ dapr uninstall -k
 # This will remove the .dapr directory present in the path <path-to-install-directory>
 dapr uninstall --runtime-path <path-to-install-directory>
 `,
-	PreRun: func(cmd *cobra.Command, args []string) {
+	PreRun: func(cmd *cobra.Command, _ []string) {
 		viper.BindPFlag("network", cmd.Flags().Lookup("network"))
 		viper.BindPFlag("install-path", cmd.Flags().Lookup("install-path"))
 	},
-	Run: func(cmd *cobra.Command, args []string) {
+	Run: func(*cobra.Command, []string) {
```

Comment:
```suggestion
	Run: func(*cobra.Command, _ []string) {
```

Reply:
Is that what you mean?
```go
func(cmd *cobra.Command, _ []string) {}
```
Here the parameter is not used, I think it might be better to follow the @go language style if possible.😄️

Reply:
Yes, we should agree on one single convention in this codebase. The other place used `_` (install.go), so recommended the same. 

Ok with using this convention - `Run: func(*cobra.Command, []string) {` in all places.